conan/internal/api/detect/detect_api.py

```diff
@@ -347,6 +347,54 @@ def detect_cppstd(compiler, compiler_version):
     return cppstd
 
 
+def default_cstd(compiler, compiler_version):
```

Comment:
In `test/unittests/client/build/cpp_std_flags_test.py` there are some tests checking the returned values of the equivalent `cppstd` default versions. Could you please create a `c_std_flags_test.py` with the equivalent tests? Thanks!